### ISSUE TITLE
error: cannot find symbol DocumentReaderCompletion

### ISSUE THREAD

**Issue by JordiGamez** [regulaforensics/DocumentReader-Android#19]
Hello,

Today, when compiling my project using Regula for more than three years, I have found this error:

```
ScreenNavigator.java:63: error: cannot find symbol
public static void openCameraScanScreen(FragmentActivity activity, DocumentReader.DocumentReaderCompletion completion) {
symbol:   class DocumentReaderCompletion
location: class DocumentReader
```

<img width="850" alt="Screenshot 2020-06-06 at 08 47 42" src="https://user-images.githubusercontent.com/10868434/83938254-05796000-a7d3-11ea-8d7e-92db1a0540b6.png">

And when I search inside the DocumentReader java class, the object DocumentReaderCompletion is not there. Instead there is an IDocumentReaderCompletion, so my question is...

I need to use **_IDocumentReaderCompletion_** instead of the current **_DocumentReaderCompletion_**?

I have searched the documentation but it's always saying to use DocumentReaderCompletion so I'm very confused at the moment because my code is saying something different than the guide.

That's in my **_grade.build_** file:

```
allprojects{
    repositories{
        maven {
            url "http://maven.regulaforensics.com/RegulaDocumentReader"
        }
    }
}
[...]
implementation 'com.regula.documentreader.mrz:core:+@aar'
implementation('com.regula.documentreader:api:+@aar') {
    transitive = true;
}
```

**Comment by vyakimchik** [comment]
Hello. Yes, you are right. We published the new version - 5.1. `DocumentReader.DocumentReaderCompletion` was changed to `IDocumentReaderCompletion`.

The demo project has been updated and the documentation too.

Link to the docs: https://docs.regulaforensics.com/android/migration-guide#completion.

**Comment by JordiGamez** [comment]
Thanks! I followed the documentation and now everything is compiling as expected.
I tried searching the IDocumentReaderCompletion in Google but nothing showed regarding this migration... 😅

**Comment by maxutolen** [comment]
> Hello. Yes, you are right. We published the new version - 5.1. `DocumentReader.DocumentReaderCompletion` was changed to `IDocumentReaderCompletion`.
> 
> The demo project has been updated and the documentation too.
> 
> Link to the docs: https://docs.regulaforensics.com/android/migration-guide#completion.

Hi. I would like to stay in version 5.0, what changes I have to make in build.gradle? Currently I'm using like this:

 ```
   implementation 'com.regula.documentreader.fullrfid:core:+@aar'
    implementation('com.regula.documentreader:api:+@aar') {
        transitive = true;
    }

```

Tried to replace +@aar to v5.0 but no luck.
Thanks
